Package.swift

```diff
@@ -0,0 +1,35 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "KeyboardKit",
+    defaultLocalization: "en",
+    platforms: [
+        .iOS(.v13)
+    ],
+    products: [
+        .library(
+            name: "KeyboardKit",
+            targets: ["KeyboardKit"]),
+    ],
+    targets: [
+        .target(
+            name: "KeyboardKitObjC",
+            path: "KeyboardKit/ObjCSources",
+            exclude: ["ResponderChainDebugging.m"],
+            publicHeadersPath: ""),
+        .target(
+            name: "KeyboardKit",
+            dependencies: ["KeyboardKitObjC"],
+            path: "KeyboardKit/SwiftSources",
+            exclude: ["UpdateLocalisedStringKeys.swift"],
```

Comment:
What I asked about above is basically like this I think:

```suggestion
            path: "KeyboardKit",
            exclude: ["ObjC", "UpdateLocalisedStringKeys.swift"],
```